docs/apis/readEndpoint.md

```diff
@@ -0,0 +1,45 @@
+## createEntity(value: { object })
```

Comment:
The name of the doc is `readEndpoint`, but title is `createEntity`. Is that normal?